### DISCUSSION TITLE
BASE_URL=变量无用

### DISCUSSION THREAD

**Discussion by taurusduan** [ChatGPTNextWeb/NextChat#2138] [Q&A]
通过镜像假设的项目,BASE_URL=填入地址无用,只能在设置里重新填入.

**Comment by Arutoria2024** [comment]
对对对，我也是遇到的这个问题，不过按照作者的描述↓
![image](https://github.com/ChatGPTNextWeb/ChatGPT-Next-Web/assets/160999144/7c39625b-8689-4d36-aad5-8f6fef846c09)
好像得先手动配置api接口代理，这个环境变量才能有用，不过具体怎么配置我也不会，如果有大佬，求教一下
